docs/adr/0020-use-internjs-testing.md

```diff
@@ -0,0 +1,69 @@
+# Use existing InternJS for functional testing in settings v2
+
+- Deciders: Vijay Budhram, Jody Heavener, Lauren Zugai
+- Date: 2020-07-17
+
+## Context and Problem Statement
+
+For over the past 5 years (at least), we have been using [InternJS](https://github.com/theintern/intern) for our functional tests in the content-server. While
+this has worked well and caught several bugs, we should reevaluate it since we are migrating to react and have an opportunity
```

Comment:
Do we have Prettier edit our MD files? There's a few weird "go to next line" spacing